retrofit-adapters/rxjava/src/main/java/retrofit/ObservableExecutionAdapter.java

```diff
@@ -0,0 +1,97 @@
+package retrofit;
+
+import java.io.IOException;
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Type;
+import rx.Observable;
+import rx.Subscriber;
+import rx.functions.Action0;
+import rx.subscriptions.Subscriptions;
+
+/**
+ * TODO docs
+ */
+public final class ObservableExecutionAdapter implements CallAdapter {
+  /**
+   * TODO
+   */
+  private static ObservableExecutionAdapter create() {
+    return new ObservableExecutionAdapter();
+  }
+
+  private ObservableExecutionAdapter() {
+  }
+
+  @Override public Type parseType(Type returnType) {
```

Comment:
parseType seems like the wrong name for this. Maybe it's something gross and long like `returnTypeToMessageType`?